docs/api.md

```diff
@@ -2316,15 +2346,6 @@ Returns SMS status for the current user.
   is in the specified country.
   <!--end-query-param-get-smsstatus-country-->
 
-##### Error responses
-
-Failing requests may be caused
-by the following errors
-(this is not an exhaustive list):
-
-* `code: 500, errno: 999`:
-  Unspecified error
```

Comment:
I don't recall deleting this, maybe it got removed as part of regenerating the docs?  In any case I don't think it's losing much of value so I didn't fix it up.

Reply:
Yep, I think some people (at least I am) are still having problems with the git hooks, so the docs aren't changing when the code changes. This deletion is linked to this change fwiw:

https://github.com/mozilla/fxa-auth-server/pull/2259/files#diff-c8e31dade53c85c9af99636f696d7266

Reply:
In fact I'm going to start using a local alias to make sure I run them every time, no idea why husky doesn't work on my machine.